src/starfleet/account_index/resolvers.py

```diff
@@ -32,17 +32,24 @@ def resolve_worker_template_accounts(loaded_template: Dict[str, Any]) -> Set[str
     return resolved_accounts
 
 
-def resolve_worker_template_account_regions(loaded_template: Dict[str, Any]) -> Dict[str, Set[str]]:
+def resolve_worker_template_account_regions(loaded_template: Dict[str, Any], org_root_check: bool = True) -> Dict[str, Set[str]]:
     """
     This will resolve the accounts/regions that a given worker template is supposed to operate on. This receives a deserialized dictionary that was based on the
     `starfleet.worker_ships.base_payload_schemas.BaseAccountRegionPayloadTemplate` (or a subclass or equivalent).
 
     This will return a Dictionary of the account ID and set of regions to operate over.
 
     This is scoped to the `STARFLEET` configuration value for `ScopeToRegions`. If that is set, then that is the maximum list of regions that can be tasked.
+
+    There is an optional parameter called `org_root_check`. This is a boolean that is default True. This is an optional if you want to use this function in
+    a worker where you define a similar config to the BaseAccountRegionPayloadTemplate, but don't have a need root checking. The AWS Config worker ship is an
```

Comment:
missing "for".  Should read "dont have a need for root checking"

Reply:
Fixed.